src/cli/helpers.ts

```
@@ -1,8 +1,11 @@
 import fs from 'node:fs/promises'
 
 import { glob } from 'tinyglobby'
+import { xfetch } from 'x-fetch'
```

ESLint

[error] Unable to resolve path to module 'x-fetch'.

(import-x/no-unresolved)